java/src/jmri/jmrix/can/cbus/eventtable/CbusBasicEventTableModel.java

```diff
@@ -65,8 +65,13 @@ public class CbusBasicEventTableModel extends javax.swing.table.AbstractTableMod
         EVENT_DAT_1, EVENT_DAT_2, EVENT_DAT_3 };
 
     public CbusBasicEventTableModel(CanSystemConnectionMemo memo) {
+        this(memo, 1);
+    }
+
+    public CbusBasicEventTableModel(CanSystemConnectionMemo memo, int initialArray) {
```

Comment:
I agree with @bobjacobsen .

Line 74:
`_mainArray = new ArrayList<>(initialArray);`

This looks like it's creating a copy of the `initialArray` instead of creating a new array with an initial capacity of `initialArray`.